test/run_elmer_test.erl

```diff
@@ -0,0 +1,63 @@
+-module(run_elmer_test).
```

Comment:
Nice, I'll refactor it a bit, so we can reuse methods from this module and the other test file.